CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+**Features**:
```

Comment:
make sure to merge master after your release... iirc git's merge feature has some quirks where this might end up in the released version. i have seen it happening. maybe it was human error during merge though

Reply:
Thanks, I would not have noticed.